src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+    <Collapser
+        id="Codestream"
+        title="CodeStream"
+    >
+    Collaboration across different development teams can be challenging, requiring developers to go back and forth between multiple tools to get anything done. New Relic's CodeStream makes it easier by allowing you to see your data, then discuss and review code, all within your IDE.
+        * See your telemetry in context, allowing developers across teams to collaborate without missing out on valuable information.
+        * Integrate with VS Code, Visual Studio, and all JetBrains editors as easily as adding an extension.
+        * View code-level metrics to fully understand production environment performance.
+        * Collaborate more easily across all your tools, such as GitHub, Slack, Teams, and email.
+    Learn more about CodeStream in our [short overview](https://newrelic.com/codestream), or see our [introduction to CodeStream](/docs/codestream/start-here/what-is-codestream/) documentation.         
+    </Collapser>
+</CollapserGroup>
+
+In addition to a suite of observability options, New Relic gives you the option to set up and report on custom data, like events and attributes to capture any data that you may want to see. You can setup these data points to be whatever you need them to be to match the needs of your organization. Check out our docs on [custom events and attributes](/docs/data-apis/custom-data/custom-events/report-custom-event-data/) to learn more.
+
+## Pricing you want, functionality you need [#pricing]
+
+Having the capabilities you need in a data monitoring solution is important, but cost often proves to be just as pivotal a factor, if not more so. New Relic offers flexible, transparant pricing focused around your knees. We believe organizations of any size should have the access to the features they need to monitor data, so we base our pricing around the amount of data and the number of users, not availability of features. 
+
+We even offer a forever-free edition, with full access to all our capabilities and features, and you only need to upgrade if your data needs grow. Test out the capabilities at no cost to see how we help you make the most of your data, then make the change to our Standard tier, designed with multiple engineers and complex workloads, or our Enterprise tier, designed for at-scale organizations with advanced security and support needs. All our editions include:
+
+* 100 GB of free data ingest as month
+* Unlimited basic users
+* Unlimited hosts and CPUs 
+* Full querying capabilities
+* Custom charts and dashboards
+* Custom alerts and notifications
+
+We know that cost can be a difficult point and that every organization has different observability needs, so we've created a number of resources to help you save costs. We have a [comprehensive overview of pricing](https://newrelic.com/pricing), as well as information on [accurately estimating your data ingest costs](https://newrelic.com/blog/nerdlog/estimate-data-cost).
```

Comment:
"We have a comprehensive overview" seems a bit too exciting for a link, perhaps simply, "See our pricing page for more on how we bill"

Reply:
Fixed!